front/src/common/HelpModal/HelpModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
+import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
+
+function HelpModal() {
+  const { t } = useTranslation('home/navbar');
+  const [helpModalText, setHelpModalText] = useState<string>('');
+
+  useEffect(() => {
+    const checkHelpMarkdown = async () => {
+      const expectedFileFirstLine = '<!-- OSRD Help Markdown File -->';
+      const defaultText = '# contact@osrd.fr';
+
+      try {
+        const response = await fetch('/help.md');
+
+        if (!response.ok) {
+          setHelpModalText(defaultText);
+          return;
+        }
+
+        const text = await response.text();
+        setHelpModalText(
+          text.startsWith(expectedFileFirstLine)
+            ? text.replace('<!-- OSRD Help Markdown File -->', '')
```

Comment:
Is it necessary to strip the comment? HTML comments should get ignored in markdown no?

Reply:
I first thought, then I discovered our markdown renderer doesn't support this

Reply:
Bummer, makes sense!